<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Format Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    
    <!-- "Convert From" Layout -->
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_from">
      <item>
       <widget class="QLabel" name="label_from">
        <property name="text">
         <string>Convert From:</string>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_from">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    
    <!-- "Convert To" Layout -->
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_to">
      <item>
       <widget class="QLabel" name="label_to">
        <property name="text">
         <string>Convert To:</string>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_to">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>

    <!-- "Select Images" Button -->
    <item>
     <widget class="QPushButton" name="pushButton_select">
      <property name="text">
       <string>Select Images</string>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>

    <!-- "Convert" Button -->
    <item>
     <widget class="QPushButton" name="pushButton_convert">
      <property name="text">
       <string>Convert</string>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>

   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
